### PR TITLE
Fix 263

### PR DESCRIPTION
**Pull request by nojaf**
Poor fix for #263.
Needs improvement, it's more of a coincidence that this works now.
@jindraivanek any ideas how to get the `range` at that point of the code?

### CODE REVIEW

**Comment by jindraivanek** [comment]
@nojaf I looked at it, and I don't see a better way than add `range` as parameter to `genMemberFlags`.

**Comment by nojaf** [comment]
Thanks @jindraivanek , I'll take a look at it.

File: src/Fantomas/CodePrinter.fs [hunk 1]
```diff
@@ -424,7 +424,34 @@ and genMemberFlags astContext = function
     | MFMember _ -> !- "member "
     | MFStaticMember _ -> !- "static member "
     | MFConstructor _ -> sepNone
-    | MFOverride _ -> ifElse astContext.IsInterface (!- "member ") (!- "override ")
+    | MFOverride _ -> ifElse astContext.InterfaceRange.IsSome (!- "member ") (!- "override ")
+        
+  and genMemberFlagsForMemberBinding astContext (mf:MemberFlags) (rangeOfBindingAndRhs:Microsoft.FSharp.Compiler.Range.range) =
```
Comment (jindraivanek) [inline src/Fantomas/CodePrinter.fs]: Indentation.

**Comment by jindraivanek** [comment]
Looks good.

**Comment by nojaf** [comment]
Thanks, I'll add some more tests to verify that it still works if you have a mix of `member` and `override` in the same type.

**Comment by nojaf** [comment]
Ready for review.

**Pull request merged**